### ISSUE TITLE
Problems with ROWID

### ISSUE THREAD

**Issue by pawelsalawa** [pawelsalawa/sqlitestudio#2840]
_(This issue was migrated from the old bug tracker of SQLiteStudio)_

Original ID from old bug tracker: 2952
Originally created at: Sat Oct  3 09:56:29 2015
Originally last updated at: Sat Oct  3 09:56:29 2015

Hello,

I receive the following error:
[09:45:45] Cannot edit this cell. Details: The query execution mechanism had problems with extracting ROWID's properly. This might be a bug in the application. You may want to report this.

I cannot use SQLite 3 DB it's equal if i create an empty DB with SQLiteStudio 3.0.6 themself or if I use an alternative SQLite application.
I searched via google many website for a solution but I don't found an helpful answer. I only found a bug inside your bug tracker tool: http://bugs.sqlitestudio.pl/?id=2874 but no answer is helpful.
Can you help me to detect the problem.

Best regards
Marcel

**Plugins loaded:**
MultiEditorTimePlugin, MultiEditorDateTimePlugin, ScriptingSql, PopulateDictionary, MultiEditorNumericPlugin, RegExpImport, XmlExport, CsvImport, PopulateSequence, PdfExport, JavaScriptHighlighterPlugin, PopulateRandom, MultiEditorTextPlugin, ScriptingQt, PopulateScript, SqliteHighlighterPlugin, Printing, DbSqlite2, ScriptingTcl, PopulateConstant, JsonExport, SqlFormatterSimplePlugin, MultiEditorBoolPlugin, HtmlExport, MultiEditorHexPlugin, PopulateRandomText, MultiEditorDatePlugin, DbPluginSqlite3, ConfigMigration, SqlEnterpriseFormatter, SqlExport, CsvExport

**Version:**
3.0.6

**Operating System:**
Windows 8.1, 64bit

**Comment by pawelsalawa** [comment]
Originally created at: Sat Oct  3 09:56:29 2015

I do some more tests.
After i start with &quot;SQLiteStudio.exe -d --debug-sql&quot; I figured out that there is a problem with:
[10:12:19.969] DEBUG:    Smart execution failed at step QueryExecutorExecute &quot;&quot; 
Using simple execution method.

I changed in the configuration from SQL Enterprise to SQL Simple and now I can change all rows/columns in my database files.
What is difference between SQL Enterprise and SQL Simple? And what I should do to fix SQL Enterprise usage?

Best Regards
Marcel
